hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/BlockPoolSlice.java

```diff
@@ -749,6 +749,8 @@ void addToReplicasMap(ReplicaMap volumeMap, File dir,
             lazyWriteReplicaMap, isFinalized, exceptions, subTaskQueue);
         subTask.fork();
         subTaskQueue.add(subTask);
+      } else if (isFinalized && ++scanBlockCnt % 10000 == 0) {
+        LOG.info((scanBlockCnt - 1) + " blocks (finalize state) have been scanned.");
```

Comment:
Please take care of concurrency issues that lead to inaccurate counting.

Reply:
The log style is not easy to view.

Reply:
ok. I will fix it.

Reply:
> The log style is not easy to view.

I have optimize it: 
![Uploading image.png…]()